src/App.css

```diff
@@ -8,12 +8,12 @@
   margin: 0;
```

Comment:
- [ ] I just noticed that you've done a great job so far on this project, and I commend your efforts :clap:. However, I'd like to address some feedback regarding the design aspect of the application. 
- [ ] While the functionalities are well-implemented, I noticed that the design does not entirely align with the specified guidelines provided by Microverse Team. In particular, I recommend focusing on refining the typography, layout, and color scheme to better match the [guidelines Design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) . By doing so, we can create a more cohesive and user-friendly experience. Your dedication to improvement is admirable, and I'm confident that with some adjustments, the final result will be outstanding. Keep up the excellent work! If you need any support or guidance, feel free to reach out. :wink:  

- [ ] Creating another video presentation with the updated design will provide valuable practice and help ensure a polished user experience. Your dedication to improvement is admirable, and I'm confident the final outcome will be exceptional. kindly fix this :point_down: 
![Screenshot from 2023-08-02 14-37-25](https://github.com/microhass/thee-worth/assets/88894525/d53a02a2-10a2-47f9-bb35-12c40cba64ae)

Choose the design among these two :point_down:  this how it should look like :point_down: 

https://github.com/microhass/thee-worth/assets/88894525/86eea6c8-0275-43cd-9ec1-5ae2a8630d53